### DISCUSSION TITLE
Config file for textcat multilabel

### DISCUSSION THREAD

**Discussion by hp52** [explosion/spaCy#7078] [Help: Coding & Implementations] [answered]
Hi,
I'm trying to do multilabel text classification with transformers and Spacy 3.0. I generated a config file with the quickstart widget, but there's no option for textcat_multilabel, just for textcat. So I'm using the textcat config file and changed the factory into "textcat_multilabel":
`[components.textcat]
factory = "textcat_multilabel"`
Is that everything I have to change or should I change something else?
Thanks for your help.

**Comment by svlandeg** [comment, marked as answer]
You're right - we're in the middle of [fixing](https://github.com/explosion/spaCy/pull/6974) this so that the docs are up-to-date as well as the quickstart.

You can definitely fix the config yourself like you're already doing though.

It's probably a good idea to replace all occurrences of "textcat" with "textcat_multilabel", including the actual name of the pipe. Then if you ever want to add a "normal" `textcat` to the same pipeline, the names won't clash.

Also, you want to set `exclusive_classes = false` in the `[components.textcat_multilabel.model]` block. That should be about it, I think.
